modules/serving-HA-components.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-config-HA.adoc
+
+[id="serving-HA-components_{context}"]
+= Knative Serving HA components
+
+Active/passive high availability (HA) is available by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
+
+You can disable HA for these components by changing the configuration of the `KnativeServing.spec.highAvailability` field.
```

Comment:
It would be good to put a heading here. Something like "Disabling High Availability". The whole section below is related to disabling HA but we only get to know that from this sentence on line 10